docs/framework/wpf/controls/listview-overview.md

```diff
@@ -69,13 +69,13 @@ The <xref:System.Windows.Controls.ListView> control provides the infrastructure
 ## Sharing the Same View Mode  
  Two <xref:System.Windows.Controls.ListView> controls cannot share the same view mode at the same time. If you try to use the same view mode with more than one <xref:System.Windows.Controls.ListView> control, an exception occurs.  
   
- To specify a view mode that can be simultaneously used by more than one <xref:System.Windows.Controls.ListView>, use templates or styles. For an example of how to define views as <xref:System.Windows.FrameworkElement.Resources%2A>, see [ListView with Multiple Views Sample](https://go.microsoft.com/fwlink/?LinkID=160013).  
+ To specify a view mode that can be simultaneously used by more than one <xref:System.Windows.Controls.ListView>, use templates or styles. For an example of how to define views as <xref:System.Windows.FrameworkElement.Resources%2A>, see [ListView with Multiple Views Sample](https://msdn.microsoft.com/library/ms771469).  
```

Comment:
A better link target for both links is <https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-3.0/ms771469(v=vs.85)>.

Reply:
Made the change based on your recommendation.